### ISSUE TITLE
Disconnects on write

### ISSUE THREAD

**Issue by georok** [ofek/csi-gcs#69]
I installed v0.7.0 as per the instructions (kubectl apply -k "github.com/ofek/csi-gcs/deploy/overlays/stable?ref=v0.7.0"
). After that I installed the "static" demo. I tested copying files between the /data directory and /tmp. All worked fine with "small" files in both directions( I tested with up to 140MB). When testing with two big files (1.3 GB and 1.4 GB) reading was fine (cp /data/1/* /tmp). When trying to copy any of the big files to the GCS volume (cp /tmp/* /data/2) the copy would fail in 10 to 20 seconds. After that df produces "df: /data: Transport endpoint is not connected".

I tried with the default PV settings as well as with limitOpsPerSec: "-1". 
I haven't tried applying limits to the demon set.

What am I missing?

thanks.

**Comment by georok** [comment]
Doing some more research I found that it was a gcsfuse issue rather than csi-gcs one. It looks like setting --max-retry-sleep=0 solved the problem. Later on, through some googling, I found that you already discussed this issue in pull request #64 and it was confirmed by @ToxicWar that it worked.

I truly appreciate the work you have done in bringing gcsfuse support to Kubernetes, but if I can make one suggestion it will be to include this information in the documentation. Technically gcsfuse in unusable without that parameter.
